src/bootstrap.tsx

```diff
@@ -687,6 +688,10 @@ const renderApp = async () => {
       i18n.changeLanguage(defaultLanguage);
     }
 
+    const printApp = appConfig?.clientConfig?.printApp;
+
+    store.dispatch(setPrintApp(printApp));
```

Comment:
Since `printApp` is optional, what happens if it's not defined?

Reply:
So far, when not defined in defaults to the defult templates setting, i tested it several times to make sure

Reply:
See comment in new review.